### DISCUSSION TITLE
Preblema con la Task 4 Lab 4.01

### DISCUSSION THREAD

**Discussion by JohanSH7** [rramosp/ai4eng.v1#286] [04 - GESTIÓN DE DATOS]
Creo que la solucion es la correcta pero no me califica el grader
![image](https://github.com/rramosp/ai4eng.v1/assets/87247883/c1cd1cc4-2dfb-4fcc-993c-337c980e9579)

notebook: https://colab.research.google.com/drive/1MU1IADhcVR90Ccg0ACvywKvHqHHZTs57?usp=sharing

**Comment by Felipe-RA** [comment]
Saludos @JohanSH7 

El problema esta en que tu código esta rompiendo el dataset aleatorizado original.

Es por esto que cuando busca la llave "place" no la encuentra.

Recuerda el buen uso de las copias para no perder información.

Quedo pendiente de cualquier duda.

Hasta pronto!
